docs/_articles/en/v3.x/troubleshooting.md

```diff
@@ -201,6 +225,9 @@ Both methods take int values of the number of milliseconds to wait. The default
 To complete the execution within the timeout period, you can also reduce the number of files sent as `--target`, which reduces the number of entry points scanned per run.
 
 
-### My `scanner:run:dfa` execution ran into this error: `java.lang.OutOfMemoryError: Java heap space`.
+### My `scanner:run:dfa` execution ran into this error: `java.lang.LimitReachedError: Java heap space`.
+
+This `LimitReachedError` is a current, known Graph Engine issue. Refer to our [recommendations](./en/v3.x/salesforce-graph-engine/working-with-sfge/#outofmemory-java-heap-space-error) to reduce the probability of experiencing the heap space error.
```

Comment:
The last part of the link needs to be changed to `#understand-limitreached-errors`.